site/app/templates/polls/ViewPollResults.twig

```diff
@@ -11,8 +11,9 @@
             data[0].x.push("{{ poll.getResponseString(answer) }}");
             data[0].y.push({{ number }});
         {% endfor %}
+        const title_text = new DOMParser().parseFromString("{{ poll.getName() }}", "text/html");
```

Comment:
If this is down in a `.twig` file why not use a PHP function to decode/unescape the poll name. You can do this here or in the `.getName` function - but I wouldn't recommend that since its in the model and there is no associated view file which would be the best place for it.

You can use https://www.php.net/manual/en/function.htmlspecialchars-decode.php if only HTML special chars are escaped (I'm not sure the actual encoding method used for polls - assuming its that from the PR description)

Reply:
The issue here is that if a PHP or Twig decoding method was used, this code would be susceptible to an XSS attack.  The method used here was done deliberately to make the string be valid JS before it is decoded.  Unless I missed something, I believe that the current method should be safe in all modern browsers.

Reply:
I don't see how using twig to decode would cause an XSS attack (unless you would use twig's raw filter but that wouldn't be the correct decoding method for this case anyways)

Reply:
Ok.  If the decoding were to be done on the server-side, how would it work such that plotly doesn't try to encode it again?  The only way I could think of was by using the `|raw` filter after un-escaping every character which is bad for the aforementioned reasons.